src/index.html

```diff
@@ -6,9 +6,61 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logoImage" class="logo">
+        <img
+          src="images/logo.png"
+          alt="moyoLogo"
+          class="moyoImg"
+          >
```

Comment:
let's fix indents
```suggestion
        <img
          src="images/logo.png"
          alt="moyoLogo"
          class="moyoImg"
        >
```

Reply:
Also, the alt attribute should be understandable for a common user. Use something like `Moyo logo` instead